streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -172,7 +172,7 @@
      * </pre>
      */
     public enum State {
-        CREATED(1, 2, 3), RUNNING(2, 3), REBALANCING(1, 2, 3), PENDING_SHUTDOWN(4), NOT_RUNNING;
+        CREATED(1, 2, 3), REBALANCING(1, 2, 3), RUNNING(1, 2, 3), PENDING_SHUTDOWN(4), NOT_RUNNING;
```

Comment:
I do not understand why `running` can be transit itself, could you describe when this could happen?